app/src/main/java/com/github/featuredetectandroid/utils/KeypointExtractionAlgorithm.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.featuredetectandroid.utils
+
+import android.util.Log
+
+private const val TAG = "KeypointExtractionAlgorithm"
+
+enum class KeypointExtractionAlgorithm {
```

Comment:
It's keypoint detection in our UI, not extraction -- let's be consistent

Reply:
fixed